fuel-coroutines/src/main/kotlin/com/github/kittinunf/fuel/coroutines/Coroutines.kt

```diff
@@ -12,28 +17,17 @@ import kotlinx.coroutines.withContext
 import java.nio.charset.Charset
 import kotlin.coroutines.CoroutineContext
 
-private suspend fun <T : Any, U : Deserializable<T>> Request.await(
-    deserializable: U,
-    scope: CoroutineContext
-): Triple<Request, Response, Result<T, FuelError>> =
-    withContext(scope) {
-        awaitResponse(deserializable)
-    }
-
-suspend fun Request.awaitByteArrayResponse(
-    scope: CoroutineContext = Dispatchers.Default
-): Triple<Request, Response, Result<ByteArray, FuelError>> =
-    await(ByteArrayDeserializer(), scope)
+suspend inline fun <T : Any, U : Deserializable<T>> Request.await(deserializable: U, scope: CoroutineContext = Dispatchers.Default): T =
```

Comment:
why exactly are we defaulting to `Dispatchers.Default` and not `Dispatchers.IO` ?

Reply:
No clue, was already the case.

Reply:
https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/-coroutine-dispatcher/

According to this blocking I/O specifically should use `.io`. Should I change it?

Reply:
yes, IO is what we should use for all coroutines in Fuel by default
IO is for IO-bound work
and Default is for CPU-bound work